lib/botRepository.js

```diff
@@ -17,17 +18,25 @@ class ChatRepository {
 
   addChat(chatId) {
     this.redis.sadd(CHATS_SET_KEY, chatId);
-  };
+  }
 
   delChat(chatId) {
     this.redis.srem(CHATS_SET_KEY, chatId);
-  };
+  }
 
   getAllChats() {
     return Rx.Observable.fromPromise(this.redis.smembers(CHATS_SET_KEY))
       .flatMap(el => el);
   }
 
+  getLastPolledDate() {
```

Comment:
:-1: would have split them in 2 repositories (you even use those 2 methods only in another class)